core/migrations/0022_auto_20240624_2118.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.0.6 on 2024-06-24 19:18
+from __future__ import unicode_literals
+
+from django.db import migrations, transaction
+from django.utils.text import slugify
+
+from core.models import Community
```

Comment:
This wont work. 
As far as I know, you don't access models this way in migrations
`Post = apps.get_model('blog', 'Post')`
https://docs.djangoproject.com/en/5.0/topics/migrations/#data-migrations

Reply:
Corrected, but as I said, almost all tests failed. Am I missing something?